Tests/ManualTests/ManualTestingApp/PointerAndMouse/PointerAndMouse.ux.uno

```diff
@@ -53,4 +53,3 @@ public partial class PointerAndMouse
         return result;
     }
 }
-
```

Comment:
I don't think it's at all a big deal in this case, especially at the end of the file. But I'll admit there's a small amount of satisfaction in pointing out white space "errors" from you specifically :) . I think it's fine!